### PR TITLE
Feat/payouts api

### LINKED ISSUE #7: Implement payouts API
- [ ] Create a payout account
- [ ] Create layout request
- [ ] Fetch details of a payout
- [ ] Cancel a payout
- [ ] Resolve bank information
- [ ] List banks

### PR DESCRIPTION
**Pull request by oneEyedSunday**
For, closes https://github.com/fluidcoins/client-go/issues/7

cc @adelowo

### CODE REVIEW

File: fluidcoins/addresses.go [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+
+type AddressesService service
+
+type Address struct {
+	ID         uuid.UUID `json:"id"`
+	Address    string    `json:"address"`
+	Coin       Coin      `json:"coin"`
+	Domain     string    `json:"domain"`
+	MerchantID string    `json:"merchant_id"`
+	// Refernece is the unique ID for this address. You can use this id to fetch the address again
```
Comment (adelowo) [inline fluidcoins/addresses.go]:
```suggestion
        // Reference is the unique ID for this address. You can use this id to fetch the address again
```

File: fluidcoins/addresses.go [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+	CreatedAt time.Time `json:"created_at"`
+	UpdatedAt time.Time `json:"updated_at"`
+}
+
+type Coin struct {
+	ID   uuid.UUID `json:"id"`
+	Code string    `json:"code"`
+
+	HumanReadableName string `json:"human_readable_name"`
+	// This is a base64 encoded image not an actual url resource
```
Comment (adelowo) [inline fluidcoins/addresses.go]:
```suggestion
        // This is a url link ( hosted by us ) to the logo of the coin.
```

Comment (adelowo) [reply fluidcoins/addresses.go]: It is not a base64 encoded image

**Review by adelowo — CHANGES_REQUESTED** [review]

File: fluidcoins/addresses.go (hunk 2 above)
Comment (oneEyedSunday) [reply fluidcoins/addresses.go]: ok, you should also perhaps update from your end (https://developers.fluidcoins.com/reference/get_address)
Screenshot 2022-03-04 at 02 23 31

Its the same for all mentions of `Image`

File: fluidcoins/addresses.go (hunk 1 above)
Comment (oneEyedSunday) [reply fluidcoins/addresses.go]: updated

File: fluidcoins/addresses.go (hunk 2 above)
Comment (oneEyedSunday) [reply fluidcoins/addresses.go]: updated

Comment (adelowo) [reply fluidcoins/addresses.go]: Ha damn, will update the docs

**Pull request merged**